sycl/cmake/modules/UnifiedRuntimeTag.cmake

```diff
@@ -1,7 +1,6 @@
-# commit 3a1b4c7b9ba952fad6f6ad36c01101bbf368347b
-# Merge: c270a6b8 264d0468
-# Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
-# Date:   Tue Jan 28 15:16:58 2025 +0000
-#     Merge pull request #2594 from kbenzie/benie/cl-core-functions-no-dlopen
-#     Fix invalid use of dlopen()
-set(UNIFIED_RUNTIME_TAG 3a1b4c7b9ba952fad6f6ad36c01101bbf368347b)
+# commit d193046de592482c47d87fdfaf92c7b8c59c9b66 (HEAD -> bump-umf-0.11, origin/bump-umf-0.11)
+# Author: Łukasz Stolarczuk <lukasz.stolarczuk@intel.com>
+# Date:   Wed Jan 22 11:06:11 2025 +0100
+# [common] Bump UMF to early 0.11 version, from main
+# It includes i.a. MacOS fix for compiler.
+set(UNIFIED_RUNTIME_TAG d193046de592482c47d87fdfaf92c7b8c59c9b66)
```

Comment:
Is this tag update necessary for the workflow changes?

Reply:
just a dummy change to trigger `sycl` changes (in detect_changes job) - replaced now with dummy changes for `unified-runtime` dir - this commit will be removed before merging, of course